src/Request/XmlRequest.php

```diff
@@ -35,7 +35,7 @@ public function arrayToXml($array, \SimpleXMLElement $xml)
                     $this->arrayToXml($value, $xml);
                 }
             } else {
-                $xml->addChild($key, $value);
+                $xml->{$key} = $value;
```

Comment:
@subhojit777 it'd be great to have a test explaining this change. And even linking to that comment as to why we append children that way.